Dockerfile.compute-node

```diff
@@ -251,7 +251,8 @@ RUN apt update &&  \
         libsfcgal1 \
         gdb && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
-    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 && \
+    mkdir -p -m 777 /neonvm/cache
```

Comment:
Given that something related to NeonVM is a little unexpected here, I'd like to see a comment here about why it's `/neonvm`. Maybe something along the lines of:

> The control plane creates compute node VMs with a tmpfs mount at `/neonvm/cache`. Manually creating this directory allows a fallback when not running in a VM.